.rubocop.yml

```diff
@@ -13,10 +13,11 @@ AllCops:
     - '**/Rakefile'
     - '**/Vagrantfile'
     - '**/db/schema.rb'
-  DisplayCopNames: true
-  TargetRubyVersion: 2.4
-  TargetRailsVersion: 5.1
+  TargetRubyVersion: 2.5
+  TargetRailsVersion: 5.2
 
+Layout/ClassStructure:
```

Comment:
This doesn't have as many rules as [ours](https://github.com/SciMed/style-guide/blob/master/samples/model.md), but it doesn't have any disagreements as far as I can tell.